### PR TITLE
Add site-header-mobile-toggler.xsl

### PR DESCRIPTION
**Pull request by alexnantel88**

### CODE REVIEW

File: workspace/utilities/site-imports.xsl [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 	<xsl:import href="com/site-header.xsl" />
 	<xsl:import href="com/site-header-logo-link.xsl" />
+	<xsl:import href="com/site-header-mobile-toggler.xsl" />
```
Comment (fhamon) [inline workspace/utilities/site-imports.xsl]: Not sure if this should be included by default in the imports

Comment (alexnantel88) [reply workspace/utilities/site-imports.xsl]: I agree, I removed it.

File: workspace/utilities/com/site-header-mobile-toggler.xsl [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+			<add class="block" />
+			<add class="relative" />
+			<add class="width-broad height-thinner" />
+		</xsl:variable>
+
+		<xsl:variable name="attr-icon-part">
+			<add class="block" />
+			<add class="absolute left" />
+			<add class="bg-color-current-color" />
+			<add class="width-full height-2px" />
```
Comment (dominic-blain) [inline workspace/utilities/com/site-header-mobile-toggler.xsl]: I don't think `height-2px` is generated. Should we use styles in this case or add a note in comments on top of the file?

Comment (dominic-blain) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: Same thing it seems for `bg-color-current-color`.

Comment (PascalPiche) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: Current color is auto generated in class-colors.less by default

Comment (PascalPiche) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: But true that height-2px is not generated by default

Comment (dominic-blain) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: > Current color is auto generated in class-colors.less by default

Oh that's where it waaaaas !

Comment (dominic-blain) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: @PascalPiche @alexnantel88  So what do you prefer for the height? 
1. Use style
2. Comment on top of file to warn of custom class
3. Let it as is

Comment (PascalPiche) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: 4. It can be a style attribute but with a xslt param for the value.

Comment (alexnantel88) [reply workspace/utilities/com/site-header-mobile-toggler.xsl]: Done !

**Comment by nitriques** [comment]
@alexnantel88 Please let me merge PR in next time, so we can avoid merge commits ;)

**Pull request merged**